### ISSUE TITLE
JSDocs and licence comments

### ISSUE THREAD

**Issue by ottiker** [jillix/flow-nodejs#220]
Write clean JSDocs comments and put a copy of the licence text in every relevant file.

**Comment by IonicaBizau** [comment]
Isn't a `LICENSE` file enough? :open_mouth: Adding the license content in every file looks to me a like  non-sense. :smile_cat:

**Comment by ottiker** [comment]
The public client files (engine.js, flow.js, etc.) are accessible in a web-inspector, though only in dev-mode. And there the licence should also be visible.

**Comment by IonicaBizau** [comment]
@adioo I am curious what will be the license then: will be Engine open-source or not? If it will be open-source, why isn't the `LICENSE` file enough? That is supposed to include everything in the project...

**Comment by ottiker** [comment]
Of course, if it would be open source, then the `LICENSE` file is enough. But we cannot open source the engine project until we have users in the jillix service. I would love to open source engine, but sadly that's how business works :worried:.

**Comment by IonicaBizau** [comment]
@adioo OK, and if it will not be open-source yet, what license would we use? Something like _this is our code, never copy/edit/distribute it!_? :smile:

**Comment by ottiker** [comment]
@IonicaBizau I think it will be a simple copyright notice. Something like `© 2015 jillix GmbH All Rights Reserved`.

**Comment by ottiker** [comment]
Meanwhile It's open-sourced! 👍 So a licence file is enough. For JSDocs there will be a separate issue.
